### ISSUE TITLE
LongCodeValue can cause a failure in search_tree

### ISSUE THREAD

**Issue by rhaxton** [ImagingDataCommons/highdicom#309]
In highdicom.sr.utils.py find_content_items()::search_tree() near line:

https://github.com/ImagingDataCommons/highdicom/blob/69281fcfe980065113eb58f4ed9407a41e4e3845/src/highdicom/sr/utils.py#L96

fetching name_code.CodeValue can fail if the ContentSequence uses either LongCodeValue or URNCodeValue instead of CodeValue.

https://dicom.innolitics.com/ciods/comprehensive-sr/sr-document-content/0040a730/0040a168/00080100

Something like:

```
if hasattr(name_code, 'CodeValue'):
    code_value = name_code.CodeValue
elif hasattr(name_code, 'LongCodeValue'):
    code_value = name_code.LongCodeValue
else:
    code_value = name_code.URNCodeValue
```

might be needed here?

**Comment by CPBridge** [comment]
Hi @rhaxton, thanks for reporting the issue. Your suggestion makes sense to me, though I haven't thoroughly thought through whether it might have any other unintended consequences.

Would you be interested in submitting a pull request with your change, and adding a test for this case?

**Comment by rhaxton-ctx** [comment]
Yes, I can try to work something up soon for this

**Comment by CPBridge** [comment]
Resolved by #310
